templates/proto/makefile.tmpl

```diff
@@ -31,36 +31,27 @@ deps-go:
 	go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
 	go get -u github.com/golang/protobuf/protoc-gen-go
 
-generate: generate-grpc {{ if .Network.Web.Enabled }}generate-web{{- end}} {{ if .Network.Http.Enabled }}generate-http{{- end}}
+generate: generate-grpc {{ if .Service.Network.Web.Enabled }}generate-web{{- end}} {{ if .Service.Network.Http.Enabled }}generate-http{{- end}}
 
-generate-grpc:	
+generate-grpc:
 	mkdir -p gen/{{ $language }}
 	protoc ${PROTO_SOURCES} --{{ $language }}_out=plugins=grpc,paths=source_relative:proto ./proto/health/*.proto
-{{- range .Services}}
```

Comment:
Related to the comment above, if this is removed how are the the service proto generation commands produced?